src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/pipeline_management/create_repository.py

```diff
@@ -17,7 +17,6 @@
 LOGGER = configure_logger(__name__)
 DEPLOYMENT_ACCOUNT_REGION = os.environ["AWS_REGION"]
 
-
```

Comment:
According to https://peps.python.org/pep-0008/#blank-lines, there need to be two blank lines before functions start. Can you add this one back?